components/engine_rocks/Cargo.toml

```diff
@@ -74,6 +74,7 @@ fail = "0.4"
 git = "https://github.com/tikv/rust-rocksdb.git"
 package = "rocksdb"
 features = ["encryption", "static_libcpp"]
+branch = "tikv-5.2"
```

Comment:
Why 5.2 instead of 5.3?

Reply:
Reuse it, since there is no changes in rust-rocksdb during this period. (We also reused the tikv-5.0 branch for v5.0/5.1 before)

Reply:
should probably call it for example `tikv-5.2+` for future branches.